contrib/craftscripts/clearblocks.js

```diff
@@ -0,0 +1,60 @@
+// Marks all blocks that have at least 3 exposed sides (adjacent to an air block)
+// and then deletes those blocks.
+// Based on jsa's MCEdit Filter clearBlocks.py
+
+importClass(Packages.com.sk89q.worldedit.world.block.BlockTypes)
+
+context.checkArgs(0, -1, "")
+
+var blocks = context.remember()
+var region = context.getSession().getRegionSelector(player.getWorld()).getRegion()
+
+var min = region.getMinimumPoint()
+var xi = region.getWidth()
+var yi = region.getHeight()
+var zi = region.getLength()
```

Comment:
I'm not sure if these matter as much, but why not name these `width`, `height`, `length`, rather than the somewhat crypting "x index", which is what I presume `xi` means.

Reply:
Variable renamed